src/shared/components/Header/index.tsx

```diff
@@ -9,8 +9,30 @@ const epflLogo = require('../../EPFL-logo.svg');
 interface InformationContentProps {
   version: string;
   githubIssueURL: string;
+  serviceVersions?: serviceVersions;
 }
 
+export  type serviceVersions = {
```

Comment:
extra space between `export` and `type`. Also by convention, our types always start with a capital letter
```suggestion
export type ServiceVersions = {
```